src/app/users/users-update/users-update.component.html

```diff
@@ -58,13 +58,26 @@
           </mat-select>
           <mat-error><planet-form-error-messages [control]="editForm.controls.level"></planet-form-error-messages></mat-error>
         </mat-form-field>
-        <label>File Upload:</label>
-        <input type="file" (change)="bindFile($event)" />
       </div>
       <div>
         <button type="submit" mat-raised-button color="primary" i18n>Update</button>
         <button type="button" mat-raised-button color="warn" (click)="goBack()" i18n>Cancel</button>
       </div>
     </form>
+    <div class="profile-upload-view">
+      <img class="profile-upload-image" src="">
+      <br>
```

Comment:
A `<br>` tag should not be used in this case.  It has a specific use for breaking lines of text in poetry & addresses.  Instead you could wrap the button in a block level (div) element.